### ISSUE TITLE
It's not possible to put generated files to a separate directory

### ISSUE THREAD

**Issue by victor-semenovich-dev** [lejard-h/chopper#443]
The generated files in my project should be placed to the separate directory "/lib/generated". It works for the source_gen package, but not for the chopper_generator. Here is my build.yaml file:

```
targets:
  $default:
    builders:
      source_gen|combining_builder:
        options:
          build_extensions:
            '^lib/{{}}.dart': 'lib/generated/{{}}.g.dart'
      chopper_generator:
        generate_for:
          - "lib/data/remote/service/**.dart"
        options:
          build_extensions:
            '^lib/{{}}.dart': 'lib/generated/{{}}.chopper.dart'
```

Is it possible to configure the build.yaml to put the generated files to the separate directory?

**Comment by techouse** [comment]
If my memory serves me right this isn't yet supported.

**Comment by techouse** [comment]
@victor-semenovich-dev I have made a PR here #444 

Would you mind checking out the repo and testing against your config?

Add this to your `pubspec.yml` to use the PR directly in your project

```yaml
dependency_overrides:
  chopper_generator:
    git:
      url: https://github.com/techouse/chopper.git
      ref: feat/issue-443-build-extensions
      path: chopper_generator
```

**Comment by victor-semenovich-dev** [comment]
@techouse Works like a charm, thank you!

**Comment by techouse** [comment]
Once #444 hets merged I'll go ahead an release a new version, however, it could take up to a few days. In the meantime just use the PR branch.

**Comment by techouse** [comment]
Released in `chopper_generator` **v6.0.2**
